examples/api/service/readme.md

```diff
@@ -66,7 +68,7 @@ Next, we are going to build our service.
 		}
 		defer con.Close()
 
-		fmt.Fprintln(con, "Hello! This is whyrusleepings awesome ipfs service")
+		fmt.Fprintln(con, "ZERO IPFS service. Nothing to see here.")
```

Comment:
sorry, i dont mean to be annoying, but since this is @whyrusleeping's example, i think he should get editorial ability to chose wording of example test. I'll defer to him on this